adafruit_sht31d.py

```diff
@@ -276,45 +283,45 @@ def mode(self, value):
         self._mode = value
 
     @property
-    def repeatability(self):
+    def repeatability(self) -> Tuple[str, bool, int]:
```

Comment:
This is pretty similar to the `mode` property above.  This returns (and the setter below takes) specific `str` values, so the type annotations should be `Literal["High", "Medium", "Low"]`.